SplunkRumWorkspace/SplunkRum/SplunkRum/SplunkRumOptions.swift

```diff
@@ -0,0 +1,171 @@
+        self.ignoreURLs = opts.ignoreURLs
+        self.spanFilter = opts.spanFilter
+        self.showVCInstrumentation = opts.showVCInstrumentation
+        self.screenNameSpans = opts.screenNameSpans
+        self.slowRenderingDetectionEnabled = opts.slowRenderingDetectionEnabled
+        self.slowFrameDetectionThresholdMs = opts.slowFrameDetectionThresholdMs
+        self.frozenFrameDetectionThresholdMs = opts.frozenFrameDetectionThresholdMs
+        self.networkInstrumentation = opts.networkInstrumentation
+        self.enableDiskCache = opts.enableDiskCache
+        self.spanDiskCacheMaxSize = opts.spanDiskCacheMaxSize
+    }
+
+    /**
+            Allows non-https beaconUrls.  Default: false
+     */
+    @objc public var allowInsecureBeacon: Bool = false
+    /**
+            Turns on debug logging (including printouts of all spans)  Default: false
+     */
+    @objc public var debug: Bool = false
+    /**
+                    Specifies additional attributes to add to every span.  Acceptable value types are Int, Double, String, and Bool.  Other value types will be silently ignored
+     */
+    @objc public var globalAttributes: [String: Any] = [:]
+
+    /**
+        Sets a value for the "environment" global attribute
+     */
+    @objc public var environment: String?
+
+    /**
+     Do not create spans for HTTP requests whose URL matches this regex.
+     */
+    @objc public var ignoreURLs: NSRegularExpression?
+
+    /**
+    Sets a filter that can modify or reject spans.  You can modify attributes of each span or return nil to indicate that that span should be dropped (never sent on the wire).
+    */
+    public var spanFilter: ((SpanData) -> SpanData?)?
+
+    /**
+     Enable span creation for ViewController Show events.
+     */
+    @objc public var showVCInstrumentation: Bool = true
+
+    /**
+     Enable span creation for screen name changes
+     */
+    @objc public var screenNameSpans: Bool = true
+
+    /**
+     Enable NetworkInstrumentation span creation for https calls.
+     */
+    @objc public var networkInstrumentation: Bool = true
+
+    /**
+     Threshold, in milliseconds, from which to count a rendered frame as slow.
+    */
+    @objc public var slowFrameDetectionThresholdMs: Double = 16.7
+
+    /**
+     Threshold, in milliseconds, from which to count a rendered frame as frozen.
+    */
+    @objc public var frozenFrameDetectionThresholdMs: Double = 700
+
+    /**
+     Enable caching created spans to disk. On successful exports the spans are deleted.
+     */
+    @objc public var enableDiskCache: Bool = false
+
+    /**
+     Threshold in bytes from which spans will start to be dropped from the disk cache (oldest first).
+     Only applicable when disk caching is enabled.
+     */
+    @objc public var spanDiskCacheMaxSize: Int64 = DEFAULT_DISK_CACHE_MAX_SIZE_BYTES
+
+    /**
+       Enable the slow rendering detection feature.
+     */
+    @objc public var slowRenderingDetectionEnabled: Bool = true
```

Comment:
It was enabled by default due to my mistake. When we have a feature flag it should be disabled by default

Reply:
ok